src/mame/mame.lst

```diff
@@ -25078,6 +25078,9 @@ x37                             //
 @source:makerbot/replicator.cpp
 replica1                        // Makerbot Replicator 1 desktop 3d printer
 
+@source:marion/ms68k.cpp
+ms68k                           // Marion Systems MS68K
+
```

Comment:
This shouldn’t really be in a project by itself.  It probably belongs in trainer (miscellaneous development systems and systems for learning computer architecture), homebrew (designs published with the intent of allowing hobbyists to build their own systems), or misc (things that don’t really fit anywhere else).

Reply:
Sounds like `trainer` is the best place for it then. Thanks!